schedule/yast/lvm_multipath_aarch64.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name:           lvm_multipath
+description:    >
+  Textmode installation test for lvm partitioning with no spearate home, on multipath with lvm validation.
+vars:
+  DESKTOP: textmode
+  MULTIPATH: 1
+  LVM: 1
+  SEPARATE_HOME: 0
+  HDDMODEL: scsi-hd
+  YUI_REST_API: 1
+schedule:
+  multipath:
+    - installation/multipath
+  guided_partitioning:
+    - installation/partitioning/select_guided_setup
+    - installation/partitioning/guided_setup/enable_lvm
+    - installation/partitioning/guided_setup/do_not_propose_separate_home
+  software:
+    - installation/installation_settings/validate_default_target
+  system_validation:
+    - console/validate_multipath
+    - console/validate_lvm
+    - console/hostname
+    - console/system_prepare
+    - console/force_scheduled_tasks
+    - shutdown/grub_set_bootargs
```

Comment:
this is `system_preparation` and the order is wrong in the original yaml, should be above validations.